src/styles/blocks/content.scss

```diff
@@ -0,0 +1,55 @@
+.page {
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $daySize*7 + $gap*6;
+  gap: $gap;
+
+  &__days {
+    display: flex;
+    box-sizing: border-box;
+    background-color: #eee;
+    border: 1px black solid;
+    
+    height: $daySize;
+    min-width: $daySize;
+
+    align-items: center;
+    justify-content: center;
+
+    transition: 0.5s;
```

Comment:
[checklist](https://github.com/mate-academy/layout_calendar/blob/master/checklist.md) 9th item, you should specify EVERY property for transition